src/app/reskin/components/stepper/deposit/status.tsx

```diff
@@ -0,0 +1,79 @@
+import { LoadingIndicator } from "../../../assets/loading-indicator";
+import { useEffect, useState } from "react";
+import { CheckCircleIcon } from "@heroicons/react/24/outline";
+
+function HyperLink({
+  href,
+  children,
+}: {
+  href: string;
+  children: React.ReactNode;
+}) {
+  return (
+    <a
+      target="_blank"
+      rel="noopener noreferrer"
+      href={href}
+      className="text-orange dark:text-dark-reskin-orange hover:underline"
+    >
+      {children}
+    </a>
+  );
+}
+
+function TxPendingStatus() {
+  return (
+    <div className="flex flex-col items-center justify-center mt-4">
+      <LoadingIndicator />
+      <div className="h-8 flex items-center text-xl leading-normal tracking-[-0.02em] text-center text-reskin-dark-gray dark:text-midGray uppercase mt-2">
+        Processing
+      </div>
+      <div>
+        ( Estimation{" "}
+        <HyperLink href="https://mempool.space/tx/4a5e1e4baab89f3a32518a88c31bc87f618f76673e2cc77ab2127b7afdeda33b">
+          5min
```

Comment:
qq: do we want the estimation to be the actual one (eg, from mempool based on the fee) or just a static text?

Reply:
we can just use static text 10 minutes I'm thinking this might be something we change later but for now MVP wise just static text might be good